server/plugin/template.go

```diff
@@ -273,7 +283,7 @@ Assignees: {{range $i, $el := .Assignees -}} {{- if $i}}, {{end}}{{template "use
 	template.Must(masterTemplate.New("pushedCommits").Funcs(funcMap).Parse(`
 {{template "user" .GetSender}} {{if .GetForced}}force-{{end}}pushed [{{len .Commits}} new commit{{if ne (len .Commits) 1}}s{{end}}]({{.GetCompare}}) to [\[{{.GetRepo.GetFullName}}:{{.GetRef | trimRef}}\]]({{.GetRepo.GetHTMLURL}}/tree/{{.GetRef | trimRef}}):
 {{range .Commits -}}
-[` + "`{{.GetID | substr 0 6}}`" + `]({{.GetURL}}) {{.GetMessage}} - {{.GetCommitter.GetName}}
+[` + "`{{.GetID | substr 0 6}}`" + `]({{.GetURL}}) {{.GetMessage}} - {{with . | commitAuthor}}{{.GetName}}{{end}}
```

Comment:
Nit: Is the `with` needed?
```suggestion
[` + "`{{.GetID | substr 0 6}}`" + `]({{.GetURL}}) {{.GetMessage}} - {{commitAuthor .}}{{.GetName}}{{end}}
```

Reply:
I think it should work with
```suggestion
[` + "`{{.GetID | substr 0 6}}`" + `]({{.GetURL}}) {{.GetMessage}} - {{commitAuthor .}}{{.GetName}}{{end}}
```

Reply:
Unfortunately, that is not valid template syntax due to imbalanced `end`:

```
$ go test -v ./server/... -run 'TestPushedCommitsTemplate.*'
?   	github.com/mattermost/mattermost-plugin-github/server	[no test files]
?   	github.com/mattermost/mattermost-plugin-github/server/testutils	[no test files]
?   	github.com/mattermost/mattermost-plugin-github/server/plugin/graphql	[no test files]
panic: template: pushedCommits:5: unexpected {{end}}

goroutine 1 [running]:
text/template.Must(...)
	/opt/homebrew/Cellar/go/1.21.6/libexec/src/text/template/helper.go:26
github.com/mattermost/mattermost-plugin-github/server/plugin.init.0()
	/Users/catalinp/src/mattermost-plugin-github/server/plugin/template.go:283 +0x4498
FAIL	github.com/mattermost/mattermost-plugin-github/server/plugin	0.493s
FAIL
```